src/AbstractWalkers.jl

```diff
@@ -0,0 +1,147 @@
+"""
+Module containing abstract definitions for walkers.
+"""
+module AbstractWalkers
+
+using AtomsBase
+using Unitful
+using ..Potentials
```

Comment:
Does this line import `Potentials` from `AtomsBase`?

Reply:
No, it imports `Potentials` locally. Any modules with `..` in front are local modules, i.e., part of `FreeBird`.